xwiki-platform-tools/xwiki-platform-tool-packager-plugin/src/main/java/org/xwiki/tool/packager/PackageMojo.java

```diff
@@ -750,6 +750,10 @@ private String getContextPath(Artifact warArtifact)
         mandatoryTopLevelArtifacts.add(this.repositorySystem.createArtifact("org.xwiki.platform",
             "xwiki-platform-user-script", getXWikiPlatformVersion(), null, "jar"));
 
+        // Default URL API implementation
+        mandatoryTopLevelArtifacts.add(this.repositorySystem.createArtifact("org.xwiki.platform",
+            "xwiki-platform-url-default", getXWikiPlatformVersion(), null, "jar"));
+
```

Comment:
why is this needed since we were supposed to already have a dep for URLs?

Reply:
You mean `xwiki-platform-url-scheme-standard` that you mentioned on the other comment?

Reply:
yes